res/controllers/common-hid-packet-parser.js

```diff
@@ -1075,6 +1087,15 @@ HIDController.prototype.processIncomingPacket = function(packet,delta) {
     }
 }
 
+HIDController.prototype.isParameterControl = function(control) {
```

Comment:
this feels like a bit of hack... maybe a property on a scaler would be a nice way to trigger setParameter vs. setValue? 

```
myscaler = function() { ... }
myscaler.useSetParameter = true;
setScaler("super", myscaler);
```

then lower down here where we lookup the scaler, 

```if (scaler !== undefined & scaler.useSetParameter) { setParameter } else { setValue }```

the downside being that you have to set the scaler on tons of controls instead of doing a prefix match